src/meta/model_v2/src/function.rs

```diff
@@ -41,6 +41,7 @@ pub struct Model {
     pub language: String,
     pub link: String,
     pub identifier: String,
+    pub body: String,
```

Comment:
Also need to add column `body` in two places in this file: https://github.com/risingwavelabs/risingwave/blob/7c84d06ff4f8b705117b862619a1f6766ca2d4c0/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L1101
https://github.com/risingwavelabs/risingwave/blob/7c84d06ff4f8b705117b862619a1f6766ca2d4c0/src/meta/model_v2/migration/src/m20230908_072257_init.rs#L710

Don't forget to remove `not_null` constraint if body is optional.